tck/features/expressions/map/Map3.feature

```diff
@@ -65,3 +65,25 @@ Feature: Map3 - Keys function
       | keys(m) | keys(null) |
       | null    | null       |
     And no side effects
+
+  Scenario Outline: [4] Using `keys()` on map with null values
+    Given any graph
+    When executing query:
+      """
+      RETURN keys(<map>) AS keys
+      """
+    Then the result should be (ignoring element order for lists):
+      | keys     |
+      | <result> |
+    And no side effects
+
+    Examples:
+      | keys                  | result          |
+      | {}                    | []              |
+      | {k: 1}                | ['k']           |
+      | {k: null}             | ['k']           |
+      | {null: null}          | ['null']        |
```

Comment:
🙈

Reply:
I see the point of adding this here, but technically the key `null` is no different from the key `k`. It is amusing, but it is potentially confusing, by making an unaware reader think that Cypher supports some kind of odd concept of `null` keys... which we don't. Maybe we should not keep this?

Reply:
Well, where are different opinions on this topic and currently `null` is allowed, so it make sense to test that it does not confuse the keys function. It is the corner cases that are interesting for testing and blow up in the wild at some point if they have not been tested.

But since the topic of reserved keyword is still controversial, I do not mind removing the example.

Reply:
I mean, the only reason you wrote this is to illustrate that `null` is treated in no special way when written in a map key position; in effect it is unrelated to what this scenario is testing for. I'm not necessarily against a `MapLiteral` feature scenario including both this literal as well as `{true: false, false: true, `1`: 1}` kind of maps, but then I would say it should be clear in the scenario description that it deals with the topic of map keys.